src/components/CommentsList.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { deleteComment } from '../api/comments';
+import { ErrorTypes } from '../constants';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comments: Comment[];
+  setErrorType: React.Dispatch<React.SetStateAction<ErrorTypes | null>>;
+  setComments: React.Dispatch<React.SetStateAction<Comment[]>>;
+};
+
+export const CommentsList: React.FC<Props> = ({
+  comments,
+  setComments,
+  setErrorType,
+}) => {
+  const handleDeleteComment = useCallback(async (commentId: number) => {
```

Comment:
useCallback is useless here, actually. It's a large topic but if you want to dive deeper, you can read more here
https://medium.com/geekculture/great-confusion-about-react-memoization-methods-react-memo-usememo-usecallback-a10ebdd3a316

Reply:
thank's a lot for the recommendation 👍